legend-sdlc-server-fs/src/main/java/org/finos/legend/sdlc/server/startup/LegendSDLCServer.java

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Goldman Sachs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package org.finos.legend.sdlc.server.startup;
+
+public class LegendSDLCServer extends BaseLegendSDLCServer<LegendSDLCServerFSConfiguration>
```

Comment:
It might be good to give this class a different name to avoid confusion with the class LegendSDLCServer in the main server module.